api/src/main/resources/_core/database/mysql/stored-procedures/sp_mamba_extract_report_metadata.sql

```diff
@@ -45,13 +45,17 @@ BEGIN
                 ELSE
 
                     SET @col_count = 1;
-                    -- SET @column_array = CONCAT('{',@column_array,'}');
+                    SET @table_name = fn_mamba_remove_quotes(@flat_table_name);
+                    SET @current_table_count = 1;
+
                     WHILE @col_count <= @column_keys_array_len
                         DO
                             SELECT fn_mamba_get_array_item_by_index(@column_keys_array, @col_count) INTO @field_name;
                             SELECT fn_mamba_json_value_by_key(@column_array,  @field_name) INTO @concept_uuid;
 
-                            SET @tbl_name = '';
+                            IF @col_count > 100 THEN
+                                SET @table_name = CONCAT(fn_mamba_remove_quotes(@flat_table_name), '_', @current_table_count);
+                                SET @current_table_count = @current_table_count;
```

Comment:
Why are you setting the `current_table_count` to its same value? I was hopping to see it changing to 2 when it reaches 300 cols, to 3 when it reaches 400 columns ....

Reply:
It was creating something like this
![image](https://github.com/UCSF-IGHS/openmrs-module-mamba-core/assets/25583672/b31cbf10-371e-456b-815c-9e3775d8a8bb)
And all that end up as different tables

Reply:
yes, that's what it will do if you not implement what i suggested